lib/repl.js

```diff
@@ -415,9 +416,11 @@ function REPLServer(prompt,
   });
 
   var sawSIGINT = false;
+  var sawCtrlD = false;
```

Comment:
Can't we use `self.editorMode` itself? Is it really necessary to have `sawCtrlD`?